docs/ref/modules/inventory-sync/api-reference.md

```diff
@@ -2,8 +2,352 @@
 
 The Inventory Sync module indexes inventory state data into dedicated indices within the Wazuh-indexer (OpenSearch). The indexed data can be retrieved using the [OpenSearch API](https://opensearch.org/docs/latest/api-reference/).
 
-For querying synchronized inventory data, use the **GET /wazuh-states-*/_search** endpoint pattern.
+For querying synchronized inventory data, use the **GET /wazuh-states-\*/\_search** endpoint pattern.
```

Comment:
Just a question, why this change? I find more clear as it was before, in my case, my terminal adds a sequence escape to the "*", but not to the _.